packages/core/src/dialects/ibmi/query.js

```diff
@@ -15,41 +15,35 @@ export class IBMiQuery extends AbstractQuery {
   }
 
   async run(sql, parameters) {
-    const stacktrace = new Error().stack;
     this.sql = sql.replace(/;$/, '');
 
-    return new Promise((resolve, reject) => {
-      const complete = this._logQuery(sql, debug, parameters);
-      this.connection.query(this.sql, parameters, (error, results) => {
+    const complete = this._logQuery(sql, debug, parameters);
 
-        if (error) {
-          const formattedError = this.formatError(error, stacktrace);
-          reject(formattedError);
-
-          return;
-        }
+    let results;
```

Comment:
Not related to this PR, but the import of `_` is unused

Reply:
Done